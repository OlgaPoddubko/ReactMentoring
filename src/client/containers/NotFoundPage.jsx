import React from 'react';

import ErrorBoundary from '../components/ErrorBoundary';
import Header from '../components/Header';
import NotFound from '../components/NotFound';
import Footer from '../components/Footer';

const NotFoundPage = () => (
      <React.Fragment>
        <ErrorBoundary >
          <Header />
          <NotFound />
          <Footer />
        </ErrorBoundary>
      </React.Fragment>
    );

export default NotFoundPage;
