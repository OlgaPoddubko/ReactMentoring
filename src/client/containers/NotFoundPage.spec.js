import React from 'react';
import NotFoundPage from './NotFoundPage';

describe('render NotFoundPage', () => {
  it('should render NotFoundPage', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
