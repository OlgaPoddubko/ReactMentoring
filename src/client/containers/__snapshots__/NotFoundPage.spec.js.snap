// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NotFoundPage should render NotFoundPage 1`] = `
<React.Fragment>
  <ErrorBoundary>
    <Header />
    <NotFound />
    <Footer />
  </ErrorBoundary>
</React.Fragment>
`;
